Source/References/ItemsHoldr-0.2.1.ts

```diff
@@ -1117,6 +1122,13 @@ module ItemsHoldr {
         }
 
         /**
+         * Toggles whether autoSave is true or false.
+         */
+        toggleAutoSave(): void {
+            this.autoSave = this.autoSave ? false : true;
```

Comment:
I'm toggling the value, so if its true, I assign it to false, otherwise it was false so now I'm making it true...yes?